.gitignore

```diff
@@ -7,3 +11,5 @@ test/dummy/db/*.sqlite3-*
 test/dummy/log/*.log
 test/dummy/storage/
 test/dummy/tmp/
+
+.byebug_history
```

Comment:
You also need to remove the file from the repo if you want git to stop tracking it